plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -17,71 +16,15 @@ import (
 	"strings"
 )
 
-// mask for masking username/password from error messages
-var mask = regexp.MustCompile(`https?:\/\/\S+:\S+@`)
```

Comment:
Right, I saw that when I merged. There were only a few lines that were different so its not big deal. I can certainly submit these changes with my next commit.